src/cogs/slash_brvns.py

```
@@ -34,4 +36,7 @@ def setup(bot):
     """
     Add cog to bot
     """
-    bot.add_cog(SlashBrvns(bot))
+    try:
+        bot.add_cog(SlashBrvns(bot))
+    except Exception as e:
```

You should never just "catch all exceptions". Especially when you aren't actually handling anything, just printing the error to the log. Catch the exception that you excpect is possible to throw.